docs: switch CLAUDE.md to wider AGENTS.md (#1852)
containous/traefik-helm-chart: AGENTS.md

> # CLAUDE.md

Reviewer: ```suggestion
# AGENTS.md
```

> This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.

Reviewer: ```suggestion
This file provides guidance to AI coding agents working with this repository.
```
